components/src/commonMain/kotlin/dev/fritz2/components/validation/validation.kt

```diff
@@ -0,0 +1,23 @@
+package dev.fritz2.components.validation
+
+import dev.fritz2.validation.ValidationMessage
+import dev.fritz2.validation.Validator
+
+data class ComponentValidationMessage(val id: String, val severity: Severity, val message: String) : ValidationMessage {
```

Comment:
I think we should add a title: String. Maybe optional. This is often needed for toasts, etc.

Reply:
Good point! I come up with the following now:
```kotlin
data class ComponentValidationMessage(
    val id: String,
    val severity: Severity,
    val message: String,
    val details: String? = null,
)
```
Hopefully this fits the most of possible use cases!